collector/__init__.py

```diff
@@ -231,18 +233,18 @@ def _instrument_setup(self, configuration: Dict[str, List[str]], instrument_name
             else:
                 instconfig[key] = ulist
 
-
+        # TODO: I expect this can be simplified.
         if any(inst in instrument_name.upper() for inst in ['IGRINS', 'MAROON-X']):
             disperser = 'XD'
        
-        return Instrument(instrument_name,disperser,instconfig)
+        return Instrument(instrument_name, disperser, instconfig)
         
     def _readzip(self, 
                  zipfile: str, 
                  xmlselect: List[str], 
-                 site: str,
-                 selection=['ONGOING', 'READY'], 
-                 obsclasses=['SCIENCE'], 
+                 site: Site,
+                 selection=frozenset(['ONGOING', 'READY']),
+                 obsclasses=frozenset(['SCIENCE']),
```

Comment:
A list would do the trick, but a list is mutable, and  has a longer search time than a frozenset. pylint was complaining about passing mutable lists to functions, and since I know these both can be treated as sets and should be immutable, I decided to make them frozensets to eliminate the warning and make the API less fragile.